### DISCUSSION TITLE
where can i find the old milvus:cpu docker images

### DISCUSSION THREAD

**Discussion by XufengXufengXufeng** [milvus-io/milvus#23399] [Q&A and General discussion]
the new install method requires docker-compose and multiple containers, which is harder to manage.

**Comment by yhmo** [comment]
All milvus released images are here: https://hub.docker.com/r/milvusdb/milvus/tags
You can search the version which you want, such as https://hub.docker.com/r/milvusdb/milvus/tags?page=1&name=2.2.0
